src/api/db/migrate/20170306084558_change_repositories_remote_project_name_to_not_null.rb

```diff
@@ -1,10 +1,5 @@
 class ChangeRepositoriesRemoteProjectNameToNotNull < ActiveRecord::Migration[5.0]
   def up
-    # We need to run src/api/db/data/20170306084550_remove_duplicate_repositories.rb first,
```

Comment:
This statement doesn't hold true if you are setting up the database from scratch